### DISCUSSION TITLE
Genial export gtsave to MS Word but colors and styling is missing.

### DISCUSSION THREAD

**Discussion by aito123** [rstudio/gt#1068] [General]
Hello I'm excited with the 0.70 release of {gt} specially with the gtsave() function exporting directly to .docx. I was trying to export the exibble table just to see the look in MS Word.

Here is the code:

```{r}

tab1<- 
  exibble %>%
  gt(rowname_col = "row", groupname_col = "group") %>%
  summary_rows(
    groups = "grp_a",
    columns = c(num, currency),
    fns = list(
      min = ~min(., na.rm = TRUE),
      max = ~max(., na.rm = TRUE)
    )) %>%
  grand_summary_rows(
    columns = currency,
    fns = list(
      total = ~sum(., na.rm = TRUE)
    )) %>%
  tab_source_note(source_note = "This is a source note.") %>%
  tab_footnote(
    footnote = "This is a footnote.",
    locations = cells_body(columns = 1, rows = 1)
  ) %>%
  tab_header(
    title = "The title of the table",
    subtitle = "The table's subtitle"
  ) %>%
  opt_stylize(style = 6, color = "cyan")

```

Which looks like this:
<img width="448" alt="image of the exibble table" src="https://user-images.githubusercontent.com/53672503/192348979-ee742fb7-391e-4470-ba6f-f6c22b7bbf66.png">

But when I export it to MS Word looks a little different:

```{r}
tab1 %>% 
  gtsave(filename = "some_gtcars.docx")
```

<img width="609" alt="image" src="https://user-images.githubusercontent.com/53672503/192349186-13bff0de-2ef5-48f3-b445-c3741f007d85.png">

All the color styling is lost which I think losses some funtionality in the table. I'm guessing also color formatting and conditional coloring is also lost when exporting to MS Word (maybe also fonts and letter sizes). 

My **question** is if the gt team is looking forward to be able to export the exact table in different formats (HTML, Word, PDF) or is a much harder task than it seems? In my opinion I think it would be amazing to preserve colors because conditional formatting in packages like gt or gtExtras could be used in the analysis when exported to Word.

That's it! I like this package and looking forward for some discussion.

Thank you,
Santiago

**Comment by a-drumm** [comment]
I can confirm that font and font size are lost as well. It always exports as Calibri, whether a font is specified in gt or a style reference docx. The xml output from the `as_word` function just has Calibri.

**Reply by chris-gaskell** [reply]
I would also really appreciate this to be a feature (I.e., appearance preserved for docx output) 😊🤞
